support-frontend/assets/pages/subscriptions-redemption/subscriptionsRedemptionReducer.js

```diff
@@ -40,7 +34,6 @@ export type Action =
   | { type: 'SET_STAGE', stage: Stage }
 
 const initialState: RedemptionFormState = {
-  corporateCustomer: getGlobal('corporateCustomer'),
   stage: getGlobal('stage') || 'form',
   userCode: getGlobal('userCode'),
```

Comment:
I wasn't quite sure the difference in purpose between corporateCustomer.redemptionCode and userCode.
Also, it didn't look like corporateCustomer ever got updated (by what was returned from the ajax endpoint when you edited the code)
In the end I decided to just use userCode everywhere as I think that's enough.

Reply:
`userCode` was the code input by the user whereas `redemptionCode` was the one returned from the server. I think what you've done is fine though